packages/contracts-periphery/contracts/universal/op-nft/Optimist.sol

```diff
@@ -0,0 +1,160 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.15;
+
+import { Semver } from "@eth-optimism/contracts-bedrock/contracts/universal/Semver.sol";
+import {
+    ERC721BurnableUpgradeable
+} from "@openzeppelin/contracts-upgradeable/token/ERC721/extensions/ERC721BurnableUpgradeable.sol";
+import { AttestationStation } from "./AttestationStation.sol";
+import { Strings } from "@openzeppelin/contracts/utils/Strings.sol";
+
+/**
+ * @title  Optimist
+ * @dev    Contract for Optimist SBT
+ * @notice The Optimist contract is a SBT representing real humans
+ *         It uses attestations for its base URI and whitelist
+ *         This contract is not yet audited
+ */
+contract Optimist is ERC721BurnableUpgradeable, Semver {
+    /**
+     * @notice The attestation station contract where owner makes attestations
+     */
+    AttestationStation public immutable ATTESTATION_STATION;
+
+    /**
+     * @notice The attestor attests to the baseURI and whitelist
+     */
+    address public immutable ATTESTOR;
+
+    /**
+     * @notice  Initialize the Optimist contract.
+     * @dev     call initialize function
+     * @param   _name  The token name.
+     * @param   _symbol  The token symbol.
+     * @param   _attestor  The administrator address who makes attestations.
+     * @param   _attestationStation  The address of the attestation station contract.
+     */
+    constructor(
+        string memory _name,
+        string memory _symbol,
+        address _attestor,
+        AttestationStation _attestationStation
+    ) Semver(0, 0, 1) {
+        ATTESTOR = _attestor;
```

Comment:
Could add checks that `_attestor` and `_attestationStation` aren't `address(0)` - i believe there was previously logic where `_attestor` couldn't be set to `address(0)` when it was ownable based. Not necessary tho